src/DataTable/DataTable.scss

```diff
@@ -24,8 +23,27 @@ $data-table-pagination-dropdown-min-width: 6rem !default;
   }
 }
 
-.pgn__bulk-actions, .pgn__table-actions {
+.pgn__bulk-actions,
+.pgn__table-actions {
+  display: flex;
+}
+
+.pgn__data-table-dataview-toggle {
+  flex: auto;
+}
+
+.pgn__data-table-actions-right {
+  display: flex;
+}
+
+.pgn__data-table-actions-right-toggle-bottom {
```

Comment:
when applied, this class pushes the iconbutontoggle to the bottom row of the actions area, only when a user asks for it using a prop